api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/PaymentCycleTest.java

```diff
@@ -30,4 +35,35 @@ void shouldFailToValidatePaymentCycleWithNoClaim() {
         //Then
         assertThat(violations).hasSingleConstraintViolation("must not be null", "claim");
     }
+
+    @Test
+    void shouldApplyVoucherEntitlement() {
+        //Given
+        PaymentCycle paymentCycle = aValidPaymentCycleBuilder()
+                .voucherEntitlement(null)
+                .totalEntitlementAmountInPence(null)
+                .totalVouchers(null)
+                .build();
+        PaymentCycleVoucherEntitlement entitlement = aValidPaymentCycleVoucherEntitlement();
+        Assertions.assertThat(entitlement.getTotalVoucherEntitlement()).isGreaterThan(0);
+        Assertions.assertThat(entitlement.getTotalVoucherValueInPence()).isGreaterThan(0);
```

Comment:
Feels very weird to have assertions in the given stage. Instead could we just create an entitlement in a way in which we're explicit about the values we care about.

Reply:
This might be affected by another PR I just created, after which `aValidPaymentCycleVoucherEntitlement` has been renamed `aPaymentCycleVoucherEntitlementWithVouchers` so we can get rid of these asserts.

Reply:
@dwybourn we can't do that here because those values are set/calculated on construction so we are just making sure the values are as expected before we start the test.